proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -292,6 +294,19 @@ message PayloadOrganizationIDPGroupAdded {
   buf.alpha.registry.v1alpha1.OrganizationRole role_override = 3;
 }
 
+message PayloadOrganizationIDPGroupUpdated {
+  // organization_id is the id of the organization with the updated IDP group.
+  string organization_id = 1;
+  // organization_name is the name of the organization with the updated IDP group.
+  string organization_name = 2;
+  // old_role_override is the role override associated with the IDP updated group. Only set if the
```

Comment:
Wouldn't these always be set, otherwise what are we auditing? I.e., and organization_id/name without a role change means nothing?

Reply:
We might go from `<nothing>` to `<something>`, in which case `old_role_override` will not be set, and vice versa.

Reply:
Ah okay, so at least one of these is guaranteed to be set.

Reply:
Yup. I could document that... Should I rephrase to use `UNSPECIFIED` instead?

Reply:
I think it's fine, I misread the comment initially.